### DISCUSSION TITLE
Cipher deprecated option

### DISCUSSION THREAD

**Discussion by speedygonzales9** [schwabe/ics-openvpn#1562] [Configuration problems] [answered]
To make issues more manageable, I would appreciate it if you fill out the following details as applicable:

# General information
1. Android 9
3. [LDPlayer9](https://www.ldplayer.net/)
4. Emulator 
5. 0.7.41 via Google Play Store 
 
# Description of the issue
Hi, 

I am using the latest version of OpenVPN for Android via Google Play Store. I have tried to connect to multiple VPN servers through VPN Gate using the provided .ovpn files. I can successfully import the files to OpenVPN for Android. However when I try to a profile, I keep getting:  `DEPRECATED OPTION: --cipher set to 'AES-128-CBC' but missing in --data-ciphers (AES-256-GCM:AES-128-GCM:CHACHA20-POLY1305). OpenVPN ignores --cipher for cipher negotiations.` Attached is a screenshot of the log: 
How can I add `AES-128-CBC` to `--data-ciphers`? 

Thank you 
# Log (if applicable)

![image](https://user-images.githubusercontent.com/40548778/206003178-d03141db-c228-4b54-8e26-5e49731d27eb.png)

# Configuration file

```
# Config for OpenVPN 2.x
# Enables connection to GUI
management /data/user/0/de.blinkt.openvpn/cache/mgmtsocket unix
management-client
management-query-passwords
management-hold

setenv IV_GUI_VER "de.blinkt.openvpn 0.7.41" 
setenv IV_SSO openurl,webauth,crtext
setenv IV_PLAT_VER "28 9 x86_64 samsung SM-G977N SM-G977N"
setenv IV_HWADDR 39:66:62:64:64:63:30
tls-cert-profile legacy
machine-readable-output
allow-recursive-routing
ifconfig-nowarn
client
verb 4
connect-retry 2 300
resolv-retry 60
dev tun
remote 219.100.37.165 1195 udp
<ca>
```

**Comment by schwabe** [comment, marked as answer]
You need to set compatbility to 2.4.0 in the basic setting of the VPN. You can manually set ciphers under the crypto options of the VPN

**Reply by speedygonzales9** [reply]
Thank you! Setting the compatibility to 2.4.0 worked.
